src/components/WalletModal/Option.tsx

```diff
@@ -62,25 +65,26 @@ const IconWrapper = styled.div`
   `};
 `
 
-type OptionProps = {
-  connection: Connection
-  activate: () => void
-  pendingConnectionType?: ConnectionType
-}
-export default function Option({ connection, pendingConnectionType, activate }: OptionProps) {
-  const isPending = pendingConnectionType === connection.type
+export default function Option({ connection }: { connection: Connection }) {
```

Comment:
BTW, while we're at it, could you create a simple snapshot test for this component to make sure appropriate state / options for active option, pending option and other pending option? 🙏

Reply:
Great idea, will do!